.github/workflows/nightly_wheel_build.yml

```diff
@@ -4,19 +4,23 @@ on:
   schedule:
     # this cron is ran every Sunday at midnight UTC
     - cron: "0 0 * * 0"
```

Comment:
I am wondering if it would be polite to update this to an odd time so we don't create a load at what might be a popular scheduling time of the week.

Reply:
See https://github.com/scikit-image/scikit-image/pull/7254.